### PR TITLE
Updated kube-state-metrics to 2.6.0

### PR DESCRIPTION
**Pull request by saidu941**
PR to uptake new image for kube-state-metrics into VZ BOM

### CODE REVIEW

File: platform-operator/verrazzano-bom.json [hunk 1]
```diff
@@ -536,8 +536,8 @@
           "name": "kube-state-metrics",
           "images": [
             {
-              "image": "kube-state-metrics",
-              "tag": "v2.4.2-20221006181839-f38a1d97",
+              "image": "kube-state-metrics-jenkins",
```
Comment (desagar) [inline platform-operator/verrazzano-bom.json:539]: -jenkins should not be used in master - only for testing.

Comment (saidu941) [reply platform-operator/verrazzano-bom.json]: Updated the PR

**Comment by saidu941** [comment]
[#3](https://build.verrazzano.io/job/verrazzano-push-triggered-acceptance-tests/job/sdosapat%252FVZ-7434-upgrade-kube-state-metrics/3)

Got the clean build for the push-triggered-tests

**Review by desagar — DISMISSED** [review]

File: platform-operator/verrazzano-bom.json (hunk 1 above)
Comment (bhatpmk) [reply platform-operator/verrazzano-bom.json]: @saidu941 Don't we need to update the helm chart ?

File: CHANGES.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ Component version updates:
 Components added:
 
 - Alertmanager v0.24.0
-- kube-state-metrics v2.4.2
+- kube-state-metrics v2.6.0
```
Comment (desagar) [inline CHANGES.md:140]: This should be moved to component version updates under v1.5.0 - the components added in 1.3.0 should not be updated. We still only added 2.4.2 back in v1.3.0 - that has not changed.

**Review by bhatpmk — APPROVED** [review]

**Pull request merged**